tests/yam/agama/agama_cli.pm

```diff
@@ -0,0 +1,42 @@
+## Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run installation through CLI with Agama
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::agama_base;
+use testapi;
+use power_action_utils qw(power_action);
+use Utils::Architectures qw(is_s390x);
+use Utils::Backends qw(is_svirt);
+
+sub run {
+    my $self = shift;
+
+    # Editing configuration
```

Comment:
you can drop both comments and create a sub to reuse code here, `agama_config_edit(substitute_command)` where substitute_command is `":\%s/jose/bernhard/g\n"`